elements/pl-hex-calculator/pl-hex-calculator.js

```diff
@@ -293,7 +293,7 @@ class OperationController {
         } else {
             window.addEventListener('DOMContentLoaded', (e) => {
                 new CalculatorController(uuid)
-            }, 'once');
+            });
```

Comment:
Note that `DOMContentLoaded` only runs once, so `{ once: true }` is not necessary.